plugins/woocommerce/client/admin/client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/test/index.js

```diff
@@ -93,6 +98,11 @@ describe( 'Tracking clicks in shippingBanner', () => {
 				getExpectedTrackingData( 'shipping_banner_create_label', false )
 			)
 		);
+
+		// Wait for any async calls to complete so we don't get `Warning: An update to ShippingBanner inside a test was not wrapped in act(...).
+		await waitFor( () => {
+			expect( acceptWcsTos ).toHaveBeenCalled();
+		} );
```

Comment:
There were many warnings when running the test suite.

```
Warning: An update to ShippingBanner inside a test was not wrapped in act(...).
```

I fixed it by waiting for acceptWcsTos to be called. When clicking the button, it will trigger the async call to accept the WCS TOS and trigger a state update so we need to wait for that to complete.

Reply:
Good one!